app/main.py

```diff
@@ -112,18 +128,12 @@ def _get_client(self):
                 _logger.info('Using MSI')
                 # refer _parse_sp_file, potentially we could have mi client id from sp
                 if self.user_assigned_identity_id != "":
-                    _logger.info('Using client_id: %s', self.user_assigned_identity_id)
```

Comment:
Please add this line back, since this means to support the case when there is multi MI binding on the same VM, so we need to explicitly config MI id to acs keyvault agent.  This is not mandatory, since if there is only single MI, we don't have to specify any id.  So this line is easier to keep the original log behavior the same.

Reply:
pushed a commit to add this line back